src/blueprint/from_config/operators/expr.rs

```diff
@@ -0,0 +1,37 @@
+use crate::blueprint::*;
+use crate::config;
+use crate::config::{Config, ExprBody, Field};
+use crate::directive::DirectiveCodec;
+use crate::try_fold::TryFold;
+use crate::valid::Valid;
+
+fn setup_field(field: &Field, expr: &ExprBody) -> Field {
+  let copy = field.clone();
```

Comment:
here we could shadow this value 
```suggestion
  let field = field.clone();
```